test/api/hb-subset-test.h

```diff
@@ -57,6 +57,11 @@ hb_subset_test_open_font (const char *font_path)
 #endif
 
   hb_blob_t* blob = hb_blob_create_from_file (path);
+  if (hb_blob_get_length (blob) == 0)
+  {
+    printf ("The test font is not found.");
+    exit (1);
+  }
```

Comment:
Adding the check here, does it look good?

Reply:
Not sure what you mean.

Reply:
The same as https://github.com/harfbuzz/harfbuzz/commit/f72b748371da2d7ce327a5d15feea46960aa7dd3#diff-7e54590da7a9eb1143ac164a13c1f300L9